README.md

```diff
@@ -1,2 +1,43 @@
-# calculator
-A simple online calculator
+# Calculator
+
+A simple online Calculator for general purposes.
+
+## Features
+
+- Add
+- Multiply
+- Subtract
+- Divide
+- Square
+- Square Root
+- Power of n
+
+## Built With
+
+- HTML
+- CSS
+- JAVASCRIPT
+
+## How to Run?
+
+To run the application you need to clone the project and run the HTML file.
+
+## Screenshot
+
+![image](https://user-images.githubusercontent.com/72201530/193838132-4e22f252-ce18-42ef-8066-b1e392d784a8.png)
+
+
+## 🚀 About Me
```

Comment:
I am not sure if we really need a "About Me" section as this project evolves to an community project. Same for the "Links" section. 

@DhanushNehru what do you think?

Reply:
Yeah that's what I told to remove the about section.

Missed to see this one, will remove it

Reply:
Yeah I told to remove the about me section

Reply:
Honestly, I am not sure what else is new on this pull request? Just blank lines?